.github/workflows/scripts/github.mjs

```diff
@@ -0,0 +1,82 @@
+/**
+ * Generates a URL to the "Artifacts" section of a Github action run
+ *
+ * Unfortunately the best we can do here is a link to the "Artifacts" section as
+ * [the upload-artifact action doesn't provide
+ * the public URL](https://github.com/actions/upload-artifact/issues/50) :'(
+ *
+ * @param {number} runId - - The ID of the Github action run
+ * @returns {string} The URL to the "Artifacts" section of the given run
+ */
+export function githubActionArtifactsUrl (runId) {
```

Comment:
Would you mind keeping this file as `comments.mjs` for now?

For some reason it looks like we've deleted and totally rewritten it

Would be good for each comment to have predefined metadata like:

1. Commit SHA
2. Date updated
3. Link to workflow run

i.e. Don't need to hide `githubActionArtifactsUrl()` behind a try/catch, add the useful info to all comments?